barretenberg/cpp/docs/src/sumcheck-outline.md

```diff
@@ -0,0 +1,5 @@
+# Sumcheck Implementation
+
+Let \f$ P_1, \ldots, P_N \f$ be multilinear polynomials in \f$ d \f$ variables. We also refer them as Honk polynomials. 
+
+Upon obtaining a round challenge, the prover algorith updates the book-keeping table captured by the variable 'PartiallyEvaluatedMultivariates' introduced in honk::sumcheck::SumcheckProver
```

Comment:
Here you would ideally want to link `PartiallyEvaluatedMultivariates` to their location in the code, same for the `SumcheckProver` from Github (or maybe also in Doxygen if it's possible, not entirely sure if it is). (Also you could use ` instead of ' for Markdown to highlight things correctly)